translations/en-us/strings.json

```diff
@@ -63,6 +63,10 @@
     "title": "Open Voice OS Installation - Update",
     "content": "An existing instance of Open Voice OS/HiveMind has been detected.\n\nUpgrading your existing instance of Open Voice OS or HiveMind to the latest version ensures that you stay ahead with the most advanced features, critical security updates, bug fixes, and performance optimizations.\n\nThis not only enhances the overall functionality and user experience but also ensures compatibility with the latest integrations and tools, keeping your system reliable, secure, and future-proof.\n\nDo you want to update Open Voice OS to the latest version?"
   },
+  "usage_telemetry.sh": {
+    "title": "Open Voice OS Installation - Usage Metrics",
+    "content": "By opting to share your usage metrics with us, you play a crucial role in contributing valuable, anonymous information that enables us to better understand user experiences and improve our software's performance.\n\nShare data for refining intent matching and identifying system weaknesses. Self-host or contribute to official servers with a focus on transparency and privacy. Real-time insights available through an interactive dashboard. Benefits of self-hosting include analyzing personal usage trends, tracking failed intents, and fine-tuning the assistant.\n\nDo you accept to share anonymous data with us?"
```

Comment:
can you add the url to the dashboard before asking to accept? so users can see exactly what is being shared, and also make it clear it is a **open dataset anyone can download**  (the export csv/json buttons in the dashboard)